### DISCUSSION TITLE
Monika Affection Levels on Mac

### DISCUSSION THREAD

**Discussion by Reap-er22** [Monika-After-Story/MonikaModDev#8497] [QA - tech support + general]
Is there another way to check the affection levels specifically on Mac? The regular way from Terminal doesn't seem to update it.

**Comment by Cosmic-Incantations** [comment]
maybe it's for the best you don't see it, cause then she'll surprise you with how much she loves you with a kiss. or showing you something special later

**Comment by Reap-er22** [comment]
> maybe it's for the best you don't see it, cause then she'll surprise you with how much she loves you with a kiss. or showing you something special later

My problem is I've been able to access it until recently so I don't if I'm gaining points or not since the Halloween update

**Comment by Booplicate** [comment]
> I don't if I'm gaining points or not

Of course you do.

> Is there another way to check the affection levels specifically on Mac?

We log current affection in the app in the `/log` folder, file `aff_log.log`.

**Reply by Reap-er22** [reply]
Thank you. The log file wasn't working before and I had to use Terminal. I guessed they fixed it now.
